### PR TITLE
Add translation for Portuguese - Brazil

### PR DESCRIPTION
**Pull request by IgorDePaula**
Add translation for Portuguese - Brazil

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 vendor/
 bin/
 composer.lock
-.idea
\ No newline at end of file
+.idea
+nbproject
```
Comment (norzechowicz) [inline .gitignore]: Those two lines should be removed becuase we can't support all IDEs. You can add those lines into your globla .gitignore. 
https://help.github.com/articles/ignoring-files/

File: src/Coduo/PHPHumanizer/Resources/translations/difference.pt-br.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+just_now:
```
Comment (norzechowicz) [inline src/Coduo/PHPHumanizer/Resources/translations/difference.pt-br.yml]: please rename this file to ``difference.pt_BR.yml``

**Comment by norzechowicz** [comment]
@IgorDePaula thanks for your work! Could you also look at https://github.com/coduo/php-humanizer/blob/master/spec/Coduo/PHPHumanizer/DateTimeSpec.php#L103 spec and add similar example for Portuguese - Brazil language?

**Comment by lightglitch** [comment]
@norzechowicz this file can also be used for Portuguese from Portugal.

**Comment by norzechowicz** [comment]
@lightglitch Portuguese from Portugal should be stored in file ``difference.pt.yml`` - there might be no difference in current translations but we can't exclude such possibility in the future.

**Comment by lightglitch** [comment]
@norzechowicz What I'm saying is that you can copy and use this file as difference.pt.yml because now there is no difference. I was thinking that making a pull request with a copy of the file was unnecessary.

**Comment by norzechowicz** [comment]
PR is more then welcome even if file is just a copy of pt_PT.yml file :) Thanks for reporing that!

**Pull request merged**